_posts/2019-01-17-naip-2018-mr-sid-county-mosaics-available.md

```diff
@@ -0,0 +1,49 @@
+
+---
+title: 'NAIP 2018 Mr. Sid County Mosaics Available'
+author:
+  display_name: Rick Kelson
+  email: rkelson@utah.gov
+date: 2019-01-17 21:13:40
+categories:
+  - Featured
+tags:
+  - naip
+  - imagery
+---
+
+The statewide 2018 National Agricultural Imagery Program (NAIP) aerial photography is now available</a>. This year the imagery is higher resolution .6 meter (60 centimeter) imagery compared to 1 meter resolution in previous years. The imagery is currently only available as 3-band Natural Color (RGB) and 3-band Color Infrared (CIR) Mr. Sid compressed county mosaics. The 4-band GeoTIFF or JP2 tiles should be available in a couple months with statewide imagery services soon to follow.
+
+[![NAIP Sample]({{"/images/NAIP2016_sm.png" | prepend:site.baseurl}} "click to enlarge")]({{"/images/NAIP2016.png" | prepend:site.baseurl}}){:.inline-text-right}]
+
+The files are not yet available from [Raster.utah.gov](https://raster.utah.gov). In the meantime use the following links to download the files for your county of interest.
+
+Beaver [Natural Color (RGB)](https://storage.googleapis.com/state-of-utah-sgid-downloads/aerial-photography/naip/naip2018/NAIP2018_Beaver_RGB.zip)[Color Infrared (CIR)](https://storage.googleapis.com/state-of-utah-sgid-downloads/aerial-photography/naip/naip2018/color-infrared/NAIP2018_Beaver_CIR.zip)
```

Comment:
i think you are going to want to make this a list using `-`'s

Reply:
willdo